tt_metal/impl/dispatch/kernels/cq_prefetch.cpp

```diff
@@ -249,7 +247,6 @@ void fetch_q_get_cmds(uint32_t& fence, uint32_t& cmd_ptr, uint32_t& pcie_read_pt
             // By here, prefetch_q_ready must be false
             // Nothing to fetch, nothing pending, nothing available, stall on host
             DEBUG_STATUS("HQW");
-            DPRINT << "fetch_q_get_cmds stall" << ENDL();
```

Comment:
Some useful dprints that we're taking off here. Is this because of code binary size?

Reply:
sadly yes.  I considered wrapping all the dprints in a macro so we could enable them just when we want them